composer.json

```diff
@@ -9,7 +9,14 @@
         "php": "^7.3.0",
         "ext-ctype": "*",
         "ext-iconv": "*",
```

Comment:
We also don't use iconv, but it is needed here because of some odd polyfill that gets installed.